htmd/parameterization/cli.py

```diff
@@ -6,80 +6,74 @@
-    parser.add_argument("--theory", help="QM Theory (default: %(default)s)", choices=["HF", "B3LYP"],
-                        default="B3LYP", dest="theory")
-    parser.add_argument("--vacuum", help="Perform QM calculations in vacuum (default: %(default)s)",
-                        action="store_false", dest="vacuum",
-                        default=True)
-    parser.add_argument("--no-min", help="Do not perform QM minimisation (default: %(default)s)", action="store_true",
-                        dest="nomin", default=False)
-    parser.add_argument("--no-esp", help="Do not perform QM charge fitting (default: %(default)s)", action="store_true",
-                        dest="noesp", default=False)
-    parser.add_argument("--no-torsions", help="Do not perform torsion fitting (default: %(default)s)",
-                        action="store_true", dest="notorsion", default=False)
-    parser.add_argument("-e", "--exec", help="Mode of execution for the QM calculations (default: %(default)s)",
-                        choices=["inline", "LSF", "Slurm"], default="inline", dest="exec")
-    parser.add_argument("--qmcode", help="QM code (default: %(default)s)", choices=["Psi4", "Gaussian"],
-                        default="Psi4", dest="qmcode")
-    parser.add_argument("--freeze-charge", metavar="A1",
-                        help="Freeze the charge of the named atom (default: %(default)s)", action="append",
-                        default=None, dest="freezeq")
-    parser.add_argument("--no-geomopt", help="Do not perform QM geometry optimisation when fitting torsions (default: "
-                                             "%(default)s)", action="store_false", dest="geomopt", default=True)
-    parser.add_argument("--seed", help="Random number generator seed (default: %(default)s)", type=int,
-                        default=20170920, dest="seed")
+import numpy as np
+
+from htmd.version import version
+from htmd.queues.localqueue import LocalCPUQueue
+from htmd.queues.slurmqueue import SlurmQueue
+from htmd.queues.lsfqueue import LsfQueue
+from htmd.queues.pbsqueue import PBSQueue
+from htmd.queues.acecloudqueue import AceCloudQueue
+from htmd.qm import Psi4, Gaussian, FakeQM
+from htmd.parameterization.ffmolecule import FFMolecule
+from htmd.parameterization.fftype import FFTypeMethod
+from htmd.parameterization.ffevaluate import FFEvaluate
+
+logger = logging.getLogger(__name__)
+
+
+def getArgumentParser():
+
+    parser = argparse.ArgumentParser(description='Acellera small molecule parameterization tool')
+
+    parser.add_argument('filename', help='Molecule file in MOL2 format')
+    parser.add_argument('-c', '--charge', type=int,
+                        help='Total charge of the molecule (default: sum of partial charges)')
+    parser.add_argument('-l', '--list', action='store_true', help='List parameterizable dihedral angles')
+    parser.add_argument('--rtf-prm', nargs=2, metavar='<filename>', help='CHARMM RTF and PRM files')
```

Comment:
This can be confusing, as it maybe not be obvious for the user how to pass it to the command-line, but since it's not an essential flag and we are fixing it, it's ok